### PR TITLE
Add to the map

### PR DESCRIPTION
**Pull request by Smagley**
add

### CODE REVIEW

File: _pins/github1.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+hithubHandle: Smagley
```
Comment (crichID) [inline _pins/github1.json]: @Smagley this should be `githubHandle:`. It looks like the g in the beginning was replaced with an h.

File: _pins/github1.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+latitude: 44.716327
+longitude: 20.459953
```
Comment (crichID) [inline _pins/github1.json]: @Smagley you will need to add three dashes `---` to the end of your file. If you need any help making these changes, please check out this video: https://vimeo.com/201011288

Comment (damonti0406) [inline _pins/github1.json]: This is a comment for @Smagley

**Comment by crichID** [comment]
Closing this as a duplicate of #9199

**Pull request closed without merging**